.github/workflows/remove-label-on-approve.yml

```
@@ -1,12 +1,12 @@
-name: Detect Pull Request Review
+name: Approval
 
 on:
-  pull_request_review:
-    types:
-      - submitted
+  pull_request_comment:
```

actionlint (1.7.4)

unknown Webhook event "pull_request_comment". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)